Platform/TigerlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -928,8 +929,11 @@ BoardInit (
       }
     }
     BuildOsConfigDataHob ();
-    if (FeaturePcdGet (PcdFusaEnabled)) {
-      FusaKeyOffProcessing();
+    FeaturesCfgData = (FEATURES_CFG_DATA *) FindConfigDataByTag(CDATA_FEATURES_TAG);

```

Comment:
can you put all these code under if (FeaturePcdGet (PcdFusaEnabled)) {} condition ?   In this way, if static build flag is off, all the related code will be excluded from compiling.

Reply:
Done.